test/runtests.jl

```diff
@@ -2,6 +2,7 @@ using Base.Test
 using DataStructures
 const IntSet = DataStructures.IntSet
 import Compat: String
+using Primes
```

Comment:
The Primes package provides the `primes` function, which is deprecated in Base right?